### PR TITLE
Bugfix/int64

### PR DESCRIPTION
**Pull request by bramstoeller**

### CODE REVIEW

File: src/power_grid_model_io/utils/json.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+Custom json encoding functionalities
+"""
+
+import json
+from typing import IO, Any
+
+import numpy as np
+
+
+class JsonEncoder(json.JSONEncoder):
```
Comment (bramstoeller) [inline src/power_grid_model_io/utils/json.py:14]: This is new.

File: src/power_grid_model_io/utils/json.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        if isinstance(o, np.integer):
+            return int(o)
+        if isinstance(o, np.floating):
+            return float(o)
+        if isinstance(o, np.ndarray):
+            return o.tolist()
+        return super().default(o)
+
+
+def compact_json_dump(data: Any, io_stream: IO[str], indent: int, max_level: int, level: int = 0):
```
Comment (bramstoeller) [inline src/power_grid_model_io/utils/json.py:29]: This is just a copy from power-grid-model, with the addition of `cls=JsonEncoder` in the `json.dump` calls.

File: tests/unit/utils/test_json.py [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+import io
+
+import numpy as np
+import pytest
+
+from power_grid_model_io.utils.json import JsonEncoder, compact_json_dump
+
+
+def test_compact_json_dump():
```
Comment (bramstoeller) [inline tests/unit/utils/test_json.py:13]: The first unit tests are copied from power-grid-model.

File: tests/unit/utils/test_json.py [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+        {"id": 9, "x": 10},
+        {"id": 11, "x": 12}
+      ]
+  }
+
+]"""
+    )
+
+
+@pytest.mark.parametrize(
```
Comment (bramstoeller) [inline tests/unit/utils/test_json.py:141]: The last two tests are new.

File: tests/unit/converters/test_tabular_converter.py [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dynamic.grid.calculation@alliander.com>
 #
 # SPDX-License-Identifier: MPL-2.0
-
+import json
```
Comment (petersalemink95) [inline tests/unit/converters/test_tabular_converter.py:4]: Two new imports in this file, but I don't see them being used

Comment (bramstoeller) [reply tests/unit/converters/test_tabular_converter.py:4]: Fair point thnx, I'll remove them. I added a unit test here, but then decided that this is not the right place to test it.

**Pull request merged**